client/jetpack-cloud/sections/pricing/jpcom-masterbar/index.tsx

```diff
@@ -133,6 +138,21 @@ const JetpackComMasterbar: React.FC< Props > = ( { pathname } ) => {
 		],
 		[ translate ]
 	);
+	const bundles = useMemo(
+		() => [
+			{
+				label: translate( 'Complete' ),
+				tagline: translate( 'The ultimate toolkit' ),
+				href: `${ JETPACK_COM_BASE_URL }/complete/`,
+			},
+			{
+				label: translate( 'Security' ),
+				tagline: translate( 'Comprehensive site security' ),
+				href: `${ JETPACK_COM_BASE_URL }/features/security/`,
+			},
+		],
```

Comment:
Would you mind having all text here? Otherwise, "Bundles" will be the only hardcoded text within the code.

```suggestion
		() => ( {
		    label: translate( 'Bundles' ),
			items: [ {
				label: translate( 'Complete' ),
				tagline: translate( 'The ultimate toolkit' ),
				href: `${ JETPACK_COM_BASE_URL }/complete/`,
			},
			{
				label: translate( 'Security' ),
				tagline: translate( 'Comprehensive site security' ),
				href: `${ JETPACK_COM_BASE_URL }/features/security/`,
			} ],
		} )
```

Reply:
Ah yes, this is a fair change to request. I'll get that updated, thank you!